packages/wallet-link/src/walletLink.ts

```diff
@@ -0,0 +1,114 @@
+/* eslint-disable object-curly-newline */
+import Sodium from 'react-native-sodium'
+import Address from '@helium/address'
+import queryString from 'query-string'
+import { getUnixTime } from 'date-fns'
+import { Keypair } from '@helium/crypto-react-native'
```

Comment:
importing keypair from crypto-react-native will limit this package to only work on react native. is there any chance we want this to work from web with `@helium/crypto`? given that we're expecting the implementer to pass in a keypair anyway, we could structure this package like we do with `@helium/transactions` where we don't import either of the crypto libraries directly, but just expect a "signable" keypair-like object to be passed in

Reply:
Great suggestion. I was trying to figure out how to do this.

Reply:
It looks like the change you made isnt working properly. When I try to run `yarn docs` I get a bunch of type errors about `Type 'Keypair' is not assignable to type 'SignableKeypair'`